### ISSUE TITLE
Enhancement

### ISSUE THREAD

**Issue by alusion** [Teknikode/Teknik#1]
Add owncloud server to services. Following the release of Version 8, owncloud is a platform that is packed with features that members can greatly benefit from. If issues with integration persist, then it is worth looking into Seafile as the next best alternative.

**Comment by KittyKatt** [comment]
IIRC, we had OC, but the old login system for Teknik had problems integrating into the OC permissions/user system. Correct?

**Comment by uncled1023** [comment]
Correct, I tried integrating the login system into owncloud and had issues.  Haven't tried it again recently.
